pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -86,6 +87,16 @@ __maybe_unused static __always_inline void protocol_classifier_entrypoint(struct
         return;
     }
 
+    // connection is TLS encrypted
+    if (bpf_map_lookup_elem(&tls_connection, &skb_tup) != NULL) {
+        if (is_tcp_termination(&skb_info)) {
+            bpf_map_delete_elem(&tls_connection, &skb_tup);
+            flip_tuple(&skb_tup);
+            bpf_map_delete_elem(&tls_connection, &skb_tup);
```

Comment:
do that in `clean_protocol_classification`

Reply:
done